iOS/Flows/Conversation/Conversation List/ConversationListViewController.swift

```diff
@@ -52,11 +52,14 @@ class ConversationListViewController: FullScreenViewController,
 
     /// A list of conversation members used to filter conversations. We'll only show conversations with this exact set of members.
     private let members: [ConversationMember]
+    /// The id of the conversation we should land on when this VC appears.
+    private let startingConversationID: ConversationID?
 
-    init(members: [ConversationMember]) {
+    init(members: [ConversationMember], startingConversationID: ConversationID?) {
         self.members = members
+        self.startingConversationID = startingConversationID
 
-        let query = ChannelListQuery(filter: .containMembers(userIds: members.userIDs),
+        let query = ChannelListQuery(filter: .containsOnlyMembers(members),
```

Comment:
I made a new filter convenience function. The standard filter returns conversations that contain ANY of the members. We want conversations that contain ONLY those members.